Updated the doc for KubeTask function (#1178)
kanisterio/kanister: docs/functions.rst

   :align: left
   :widths: 5,5,5,15

>    `namespace`, No, `string`, namespace in which to execute (if not mentioned will create pod under controller namespace)

Reviewer: ```suggestion
   `namespace`, No, `string`, namespace in which to execute (if not mentioned the pod will be created in Controller's namespace)
```

Author: Done

>    `namespace`, No, `string`, namespace in which to execute (if not mentioned will create pod under ``kanister`` namespace)

Reviewer: Let's also mention where the pod will be created if we don't specify the namespace.

Author: Wrote `(if not mentioned will create pod under ``kanister`` namespace)` in the arg description. Do we need to add more context or update this line?

Reviewer: aah, I missed that I think. Let's use controller namespace instead of kanister namespace.

Author: Cool. I'll make this update. Thanks. 👍🏼

>    `namespace`, No, `string`, namespace in which to execute (if not mentioned the pod will be created in controller's namespace)

Reviewer 2: ```suggestion
   `namespace`, No, `string`, namespace in which to execute (if not specified, the pod will be created in controller's namespace)
```

>    `namespace`, No, `string`, namespace in which to execute (if not specified, the pod will be created in controller's namespace)

Reviewer 2: ```suggestion
   `namespace`, No, `string`, namespace in which to execute (the pod will be created in controller's namespace if not specified)
```
